app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -126,6 +128,14 @@ class MembersFragment : BaseFragment() {
                                 adapter = rvAdapter
                                 isRecyclerView = true
                             }
+                        } else {
+                            if (!filterMap["location"].isNullOrEmpty()) {
+
+                                if (!membersViewModel.userList.any { (it.location)?.contains(filterMap["location"].toString(), ignoreCase = true) == true }) {
```

Comment:
Yup code looks much cleaner. Just a questions. Why have you added the `toString()` class here? will something like this not work? 
```suggestion
                                if (!membersViewModel.userList.any { (it.location)?.contains(filterMap["location"]!! , ignoreCase = true) == true }) {
```